data/dapps/silo-finance.json

```
@@ -0,0 +1,11 @@
+{
+  "aliases": {
+    "silo-finance-v1-rdnt": {
+      "chains": ["arbitrum"],
+      "title": "Silo Finance RDNT market",
+      "description": "The dAPI proxies specific for Silo Finance RDNT market on Arbitrum. Proxies for other Silo Finance markets should use different configuration."
```

Alternatively:
```suggestion
      "description": "Silo Finance consists of multiple independent markets. Each market should use separate OEV proxies. This OEV proxy is intended to be used only for Silo Finance RDNT market on Arbitrum."
```

Btw. using the term `market` because they also use that on their page -> https://app.silo.finance/ (although they are referring to these markets as silos elsewhere).

---

`Only to be used for the RDNT market.` I don't see it omitting any information from the above. A URL would further clarify things but that's because multiple aliases need their own `homepageUrl` fields.

---

> `Only to be used for the RDNT market. `

Changed.

> A URL would further clarify things but that's because multiple aliases need their own homepageUrl fields.

Yeah, that would further help to distinguish those, and we could have something like this:
```jsonc
{
 "aliases": {
   "silo-finance-v1-rdnt": {
     "title": "Silo Finance RDNT market",
     "description": "Only to be used for the RDNT market.",
     "chains": {
       "arbitrum": {
         "homepageUrl": "https://app.silo.finance/silo/0x19d3F8D09773065867e9fD11716229e73481c55A"
       }
     }
   }
 }
}
```

the problem with that is that such URL is not always available (for example [Moonwell](https://moonwell.fi/markets) doesn't have an URL for a specific chain deployment). We could special case this, but I think showing only the description is good enough.

---

Yeah, I wouldn't change the schema further. The `dapis.json` approach is to use such links in the description where applicable.